site/helm-charts.md

```diff
@@ -0,0 +1,513 @@
+```
+helm get values sample-weblogic-operator1
+```
+
+Show all of the values your operator Helm release is using:
+```
+helm get values --all sample-weblogic-operator1
+```
+
+List the Helm releases that have been installed in this Kubernetes cluster:
+```
+helm list
+```
+
+Get the status of the operator Helm release.
+```
+helm status sample-weblogic-operator1
+```
+
+Show the history of the operator Helm release:
+```
+helm history sample-weblogic-operator1
+```
+
+Roll back to a previous operator Helm release, in this case release 1:
+```
+helm rollback sample-weblogic-operator1 1
+```
+
+Change one or more values in the operator Helm release. In this example, the --reuse-values flag indicates that previous overrides of other values should be retained:
+```
+helm upgrade \
+  --reuse-values \
+  --set "domainNamespaces={sample-domains-ns1}" \
+  --set "javaLoggingLevel:FINE" \
+  --wait \
+  sample-weblogic-operator1 \
+  kubernetes/charts/weblogic-operator
+```
+
+## Operator Helm Chart values.yaml details
+
+This section describes the details of the operator Helm Chart's values.yaml file.
+
+### Overall Operator information
+
+#### serviceAccount
+
+Specifies the name of the service account in the operator's namespace that the operator will use to make requests to the Kubernetes API server. The user is responsible for creating the service account.
+
+Defaults to "default".
+
+Example:
+```
+serviceAccount: "weblogic-operator"
+```
+
+#### javaLoggingLevel 
+
+Specifies the level of Java logging that should be enabled in the operator. Valid values are:  "SEVERE", "WARNING", "INFO", "CONFIG", "FINE", "FINER", and "FINEST".
+
+Defaults to "INFO"
+
+Example:
+```
+javaLoggingLevel:  "FINE"
+```
+
+### Creating the operator pod
+
+#### image 
+
+Specifies the docker image containing the operator code.
+
+Defaults to "weblogic-kubernetes-operator:2.0".
+
+Example:
+```
+image:  "weblogic-kubernetes-operator:LATEST"
+```
+
+#### imagePullPolicy
+Specifies the image pull policy for the operator docker image.
+
+Defaults to "IfNotPresent"
+
+Example:
+```
+image:  "Always"
+```
+
+#### imagePullSecrets
+Contains an optional list of Kubernetes secrets, in the operator's namepace, that are needed to access the registry containing the operator Docker image. The user is responsible for creating the secret. If no secrets are required, then omit this property.
+
+Example:
+```
+imagePullSecrets:
+- name: "my-image-pull-secret"
+```
+
+### WebLogic Domain management
+
+#### domainNamespaces
+
+Specifies a list of WebLogic Domain namespaces which the operator manages. The names must be lower case. The user is responsible for creating these namespace.
+
+This property is required.
+
+Example 1: In the configuration below, the operator will monitor the "default" Kubernetes namespace.
+```
+domainNamespaces:
+- "default"
+```
+
+Example 2: In the configuration below, the Helm installation will manage "namespace1" and "namespace2":
```

Comment:
Not sure if we should mention it, but yaml supports two different array syntaxes:

domainNamespaces: [ "domain1", "domain2" ]

domainNamespaces:
  - "domain1"
  - "domain2"

The customer can choose either.  However, I think the latter is more common.